pkg/ui/src/views/statements/planView.tsx

```diff
@@ -16,115 +16,274 @@ import _ from "lodash";
 import React from "react";
 
 import { cockroach } from "src/js/protos";
-import { intersperse } from "src/util/intersperse";
 import IExplainTreePlanNode = cockroach.sql.IExplainTreePlanNode;
-import ExplainTreePlanNode_IAttr = cockroach.sql.ExplainTreePlanNode.IAttr;
+import IAttr = cockroach.sql.ExplainTreePlanNode.IAttr;
+import {ToolTipWrapper} from "oss/src/views/shared/components/toolTip";
```

Comment:
```suggestion
import { ToolTipWrapper } from "src/views/shared/components/toolTip";
```

Nit: intellij likes to put in `oss`, yet our convention is to just go from `src`. (why they both actually work though is beyond me… webpack is complicated)

Reply:
appreciate the explanation :) - will fix.